oomph/typer.py

```diff
@@ -221,7 +225,14 @@ def do_expression(self, ast: uast.Expression) -> tast.Expression:
                 return self.create_special_call("bool_true", [])
             if ast.varname == "false":
                 return self.create_special_call("bool_false", [])
-            return tast.GetVar(self.variables[ast.varname], ast.varname, ast.lineno)
+            if "::" in ast.varname:
+                module_id, name_in_module = ast.varname.split("::")
+                path = self.file_typer.imports[module_id]
+                the_type = self.file_typer.module_var_types[path][name_in_module]
```

Comment:
Can this be changed to be less nested, perhaps `foo[(path, name_in_module)]` instead of `foo[path][name_in_module]`?